### DISCUSSION TITLE
Firefox 9.0.2.3 missing Options screen for entering license?

### DISCUSSION THREAD

**Discussion by barnold12** [aclap-dev/video-downloadhelper#813] [Help and Q&A]
### Your question

Under Firefox Add-Ons/Extensions, the 3 dot menu for videodownloadhelper now only shows Remove, Report, and Manage. 

Previously it also had Options. 

Where do we enter license information without Options?

### Addon & CoApp version and details

9.0.2.3

**Comment by GrampaWildWilly** [comment]
![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/7ca49273-aa26-4b43-9035-e3798e7952cf)
![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/19712712-b3a1-48cd-82cd-83a377c67856)
![#06](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f16b4651-4af5-464f-9d2b-4881fd0d3465)
